Drivers/seds/PioHwasComponent.xml

```diff
@@ -102,13 +102,13 @@
                   <VariableRef variableRef="pioVar.mPortAddress"></VariableRef>
                   <VariableRef variableRef="pioVar.mPortCodrAddress" />
                 </MathOperation>
-                <Assignment outputVariableRef="pioVar.mPortOdsrAddress">
-                  <Value value="56" />
+                <Assignment outputVariableRef="pioVar.mPortPdsrAddress">
+                  <Value value="60" />

```

Comment:
Why this change?

Reply:
This address was wrong. That's why I have changed it. It was the first SEDS where the PIO output was read, that's why this bug became apparent.